### ISSUE TITLE
How to include the source files in my project? (v0.8.x)

### ISSUE THREAD

**Issue by digitaldonkey** [WalletConnect/walletconnect-monorepo#38]
When I use walletconnect-web3-provider as stated in the readme, I will get a compiled and minified version in my code. Which is:

* hard to debug
* created a overhead in my own module. I'm using webpack myself and there is the nice codeSplitting feature I want to take advantage of. E.g. the buffer module is part of Web3 and part of walletconnect. If I can include the sources in my build webpack would nicely put the chunks together and I wouldn't force the user to load the same code twice. 

I played around with installing walletconnect-monorepo directly and importing like 
`import WalletConnectProvider from 'walletconnect-monorepo/packages/walletconnect-web3-provider/src'`

Bu this seems pretty strange and I currently don't know how to automatically do the the stuff done by `npm run bootstrap` and `npm run check-packages`

**Comment by digitaldonkey** [comment]
You de-minified the compiled code in 0.78 but that actually doesn't solve anything and could be archived in Crome console too by using pretty print. 

What you need is the actual source code in the subrepo created from the monorepo with sourcemaps pointing there. 

Otherwise debugging WalletConnect is a pain.

**Comment by digitaldonkey** [comment]
The only way I can debug the code unminified within my app is by cloning `walletconnect-monorepo`, run the [dev-workflow](https://github.com/WalletConnect/walletconnect-monorepo/tree/v0.7.x#development-workflow) and applying the following. 
My actual app dependencies is supposed to be `npm install walletconnect-web3-provider` only as shown in [Readme](https://github.com/WalletConnect/walletconnect-monorepo/tree/v0.7.x#for-web3-provider-web3js).

```diff
diff --git a/package.json b/package.json
index 66389b6..1fdd69f 100644
--- a/package.json
+++ b/package.json
@@ -1,5 +1,6 @@
 {
   "name": "walletconnect-monorepo",
+  "version": "0.7.8",
   "description": "Monorepo for WalletConnect Javascript Libraries",
   "private": true,
   "keywords": [
diff --git a/packages/walletconnect-web3-provider/src/index.js b/packages/walletconnect-web3-provider/src/index.js
index 7d48c32..9b2de54 100644
--- a/packages/walletconnect-web3-provider/src/index.js
+++ b/packages/walletconnect-web3-provider/src/index.js
@@ -1,6 +1,6 @@
 import ProviderEngine from 'web3-provider-engine'
 import RpcSubprovider from 'web3-provider-engine/subproviders/rpc'
-import WalletConnectSubprovider from 'walletconnect-web3-subprovider'
+import WalletConnectSubprovider from 'walletconnect-monorepo/packages/walletconnect-web3-subprovider/src'
 
 export default function(opts) {
   const bridgeUrl = opts.bridgeUrl || null
diff --git a/packages/walletconnect/src/index.js b/packages/walletconnect/src/index.js
index 86f9c08..c184af5 100644
--- a/packages/walletconnect/src/index.js
+++ b/packages/walletconnect/src/index.js
@@ -1,6 +1,7 @@
 /* global window */
 
-import { Connector } from 'js-walletconnect-core'
+// import { Connector } from 'js-walletconnect-core'
+import  { Connector } from 'walletconnect-monorepo/packages/walletconnect-core/src'
 
 const localStorageId = 'wcsmngt'
 let localStorage = null
```

**Comment by pedrouid** [comment]
Hi @digitaldonkey, I have just published v1.0.0-beta.1 libraries to NPM which now include source files and sourcemaps for each package
